### ISSUE TITLE
IKEA LED2106R3 TRADFRI bulb GU10 345lm update 1.10.1

### ISSUE THREAD

**Issue by aSilentSniper** [ioBroker/ioBroker.zigbee#2060]
Hey seit dem update auf die oben genannte version sieht das device leider so aus funktioniert aber normal.

Edit: ich vermute es liegt daran das z2m jetzt 2 Devices hat statt eins.

vorher: 

https://www.zigbee2mqtt.io/devices/LED2005R5_LED2106R3.html

jetzt:

https://www.zigbee2mqtt.io/devices/LED2106R3.html

https://www.zigbee2mqtt.io/devices/LED2005R5.html

![image](https://github.com/ioBroker/ioBroker.zigbee/assets/3136108/58c54265-18c7-4c31-827c-ee7e8fd8b2d4)
![image](https://github.com/ioBroker/ioBroker.zigbee/assets/3136108/b7b268dc-43a3-4a38-85fa-3c50b2addfcf)

**Comment by arteck** [comment]
zeigmal die hier
![grafik](https://github.com/ioBroker/ioBroker.zigbee/assets/6681528/381ae332-78f6-454f-8f3f-34915989cc2f)

**Comment by aSilentSniper** [comment]
![image](https://github.com/ioBroker/ioBroker.zigbee/assets/3136108/e7ba2772-c20b-4827-a6df-fba2e14b77fe)

habe oben gerade nen edit mit meiner vermutung gemacht

**Comment by arteck** [comment]
klickmal hier
![grafik](https://github.com/ioBroker/ioBroker.zigbee/assets/6681528/db23b327-f66d-465e-a2d2-853ae08ad863)

wo landest du

**Comment by aSilentSniper** [comment]
auf dem oberen link bei mir im nichts
https://www.zigbee2mqtt.io/devices/LED2005R5_LED2106R3.html

![image](https://github.com/ioBroker/ioBroker.zigbee/assets/3136108/9b36f252-bc2a-4f2d-8b67-f0586fba955c)

**Comment by arteck** [comment]
daran liegts es

**Comment by aSilentSniper** [comment]
ja sind jetzt 2 einzelene devices bei z2m

https://www.zigbee2mqtt.io/devices/LED2106R3.html

https://www.zigbee2mqtt.io/devices/LED2005R5.html

**Comment by uwe5** [comment]
Ja, das die Bilder aus den Internet gezogen werden, ist echt unschön. Das Problem hatte ich auch, denn mein iobroker darf nicht ins Internet (nur ab und zu wenn Updates eingespielt werden). Früher waren die Bilder direkt im Adapter, und die Bilder aus https://www.zigbee2mqtt.io/devices/* sind nicht das wahre, da kein PNG bzw. die Transparenz in den Bildern fehlt.
Damit die Abhängigkeit auf eine Internetverbindung nicht notwendig ist, habe ich im Pfad /opt/iobroker/iobroker-data/files/zigbee.admin/img/ die entsprechenden Bilder hinterlegt. Dazu musst du aber auch die Datei /opt/iobroker/node_modules/iobroker.zigbee/lib/devices.js ergänzen.

**Comment by CimCup** [comment]
Ja, es wäre echt schön, wenn man individuell Bilder im Adapter ersetzen könnte, da einige Bilder von zigbee2mqtt manchmal nicht zu den ensprechenden Produkten passen (Ich nehme an, dass einige Hersteller einfach die gleiche Firmware für verschiedene Produkte nimmt) oder sind schlecht bearbeitet sind.
Diese Zusatzfunktion im Adapter wäre echt hilfreich!

Gruß
CimCup
